sw/host/opentitanlib/src/io/uart.rs

```diff
@@ -92,6 +92,10 @@ pub trait Uart {
         Ok(())
     }
 
+    fn set_break(&self, _enable: bool) -> Result<()> {
+        Err(UartError::GenericError("break unsupported".into()).into())
```

Comment:
nit: Other traits in this directory use `Err(TransportError::UnsupportedOperation.into())` in their default implementations.

Reply:
Fixed.